New Dapr Scenario: Deploy an application using the Dapr cluster extension (#1295)
microsoft/azure_arc: docs/azure_arc_jumpstart/azure_arc_k8s/developer/_index.md

---
type: docs
title: "Developer Use Cases"
linkTitle: "Developer Use Cases"
weight: 12
description: >-

>   Once you have Kubernetes clusters projected into Azure with Azure Arc, you can start to use native Azure tooling to manage the clusters as native Azure resources. The following scenarios show examples of using developer-centric tools such as Dapr.

Reviewer: ```suggestion
  Once you onboarded your Azure Arc-enabled Kubernetes clusters, you can start to use native Azure tooling to manage the clusters as native Azure resources. The following scenarios show examples of using developer-centric use-cases.
```

Reviewer: I prefer not to be explicit about Dapr
